bfcfg

```diff
@@ -648,29 +655,157 @@ sys_cfg_one_byte()
+  else
+    internal_cpu_model_str="UNKNOWN"
+  fi
+
+  if [ "${host_priv_level}" -eq 0 ]; then
+    host_priv_level_str="PRIVILEGED"
+  elif [ "${host_priv_level}" -eq 1 ]; then
+    host_priv_level_str="RESTRICTED"
+  else
+    host_priv_level_str="UNKNOWN"
+  fi
+
+  if [ "${nic_mode}" -eq 0 ]; then
+    nic_mode_str="DPU_MODE"
+  elif [ "${nic_mode}" -eq 1 ]; then
+    nic_mode_str="NIC_MODE"
+  else
+    nic_mode_str="UNKNOWN"
+  fi
+
+  echo "sys: INTERNAL_CPU_MODEL=${internal_cpu_model_str}"
+  echo "sys: HOST_PRIV_LEVEL=${host_priv_level_str}"
+  echo "sys: NIC_MODE=${nic_mode_str}"
+
+  rm -f ${tmp_file}
+}
+
+sys_cfg_dump_pass_settings()
+{
+  local tmp_file=/tmp/.bfcfg-sysfs-data
+  local default_pass_policy
+
+  if [ ${dump_mode} -eq 0 ]; then
+    return
+  fi
+
+  if [ ! -e "${pass_settings_sysfs}" ]; then
+    log_msg "sys: failed to find the ${pass_settings_sysfs} EFI variable"
+    return
+  fi
+
+  # shellcheck disable=SC2216
+  yes | cp -f ${pass_settings_sysfs} ${tmp_file}
+  default_pass_policy=$(hexdump -s 4 -n 1 -e '/1 "%d" "\n"' ${tmp_file})
+  echo "sys: UEFI_DEFAULT_PASS_POLICY_ENABLE=${default_pass_policy}"
+
+  rm -f ${tmp_file}
+}
+
 #
 # This function writes to the BfSysCfg UEFI variable directly.
 # Below are the offsets defined in UEFI which is 4(fixed header) plus the offset
 # of the variable within the BfSysCfg struct. These offsets are not supposed to
 # change in order to be backward compatible with previous releases.
-#   VARIABLE(Name)       OFFSET(Byte)  SIZE(Byte)
-#   SYS_ENABLE_SMMU      24            1
-#   SYS_DISABLE_SPMI     25            1
-#   SYS_ENABLE_2ND_EMMC  26            1
-#   SYS_BOOT_PROTECT     27            1
-#   SYS_ENABLE_SPCR      32            1
-#   SYS_DISABLE_PCIE     33            1
-#   SYS_ENABLE_OPTEE     34            1
-#   SYS_ENABLE_I2C0      36            1
-#   SYS_DISABLE_FORCE_PXE_RETRY  37    1
-#   SYS_ENABLE_BMC_FIELD_MODE    38    1
+#   VARIABLE(Name)               OFFSET(Byte)  SIZE(Byte)
+#   SYS_ENABLE_SMMU              24            1
+#   SYS_DISABLE_SPMI             25            1
+#   SYS_ENABLE_2ND_EMMC          26            1
+#   SYS_BOOT_PROTECT             27            1
+#   SYS_ENABLE_SPCR              32            1
+#   SYS_DISABLE_PCIE             33            1
+#   SYS_ENABLE_OPTEE             34            1
+#   SYS_ENABLE_I2C0              36            1
+#   SYS_DISABLE_FORCE_PXE_RETRY  37            1
+#   SYS_ENABLE_BMC_FIELD_MODE    38            1
+#   SYS_LARGE_ICMC_SIZE          39            4
+#   SYS_DEFAULT_LARGE_ICMC_LOCK  43            1
+#   SYS_DISPLAY_FLAGS            44            1
+#   SYS_CE_THRESHOLD             45            4
+#   SYS_ENABLE_HEST              49            1
+#   SYS_L3_CACHE_PART_LEVEL      50            2
 #
 sys_cfg()
 {
   local tmp_file=/tmp/.bfcfg-sysfs-data
 
   has_sys_cfg=0
 
+  if [ ${dump_mode} -eq 1 ]; then
+    sys_cfg_dump_bf_modes
+    sys_cfg_dump_pass_settings
+  fi
+
```

Comment:
This block seems not related to UEFI SysConfig. Move it to 1425-1430 block below?

Reply:
I had moved these two functions into sys_cfg() based on feedback from @kblaiech - I think the code has moved around so lines 1425-1430 may not be pointing to the same place anymore. Were you suggesting to call at the end of bfcfg or somewhere else?